src/tests/pgp-parse.c

```diff
@@ -224,3 +228,246 @@ pgp_compress_roundtrip(void **state)
         }
     }
 }
+
+static bool
+setup_keystore_1(rnp_test_state_t *state, rnp_t *rnp)
+{
+    rnp_params_t params = {0};
+    bool         res = true;
+    char         path[PATH_MAX] = {0};
+
+    // IO
+    pgp_io_t pgpio = {.errs = stderr, .res = stdout, .outs = stdout};
+    rnp->io = malloc(sizeof(pgp_io_t));
+    if (!rnp->io) {
+        return false;
+    }
+
+    memcpy(rnp->io, &pgpio, sizeof(pgp_io_t));
+    assert(state->data_dir);
+
+    paths_concat(path, sizeof(path), state->data_dir, "keyrings/1/pubring.gpg", NULL);
+    params.pubpath = strdup(path);
+    paths_concat(path, sizeof(path), state->data_dir, "keyrings/1/secring.gpg", NULL);
+    params.secpath = strdup(path);
+    if (!params.pubpath || !params.secpath) {
+        res = false;
+        goto end;
+    }
+    params.ks_pub_format = RNP_KEYSTORE_GPG;
+    params.ks_sec_format = RNP_KEYSTORE_GPG;
+    if (rnp_init(rnp, &params) != RNP_SUCCESS) {
+        res = false;
+        goto end;
+    }
+
+    // Load keys
+    if (!rnp_key_store_load_keys(rnp, true)) {
+        res = false;
+        goto end;
+    }
+
+    // Set password
+    rnp->passphrase_provider.callback = string_copy_passphrase_callback;
+    rnp->passphrase_provider.userdata = /*unconst*/ (char *) KEYRING_1_PASSWORD;
+
+end:
+    rnp_params_free(&params);
+    return res;
+}
+
+void
+test_repgp_decrypt(void **state)
+{
+    rnp_t     rnp = {0};
+    rnp_ctx_t ctx = {0};
+
+    rnp_test_state_t *rstate = *state;
+
+    uint8_t out_buf[1024] = {0};
+    char    input_file[1024] = {0};
+    size_t  out_buf_size = sizeof(out_buf);
+    uint8_t in_buf[1024] = {0};
+    size_t  in_buf_size = sizeof(in_buf);
+    char    plaintext[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890\n";
+
+    /* -------------------------------------------------------------------------
+        Setup keystore
+         This text was encrypted with keys stored in keyrings/1/..
+       -------------------------------------------------------------------------*/
```

Comment:
Can you use one of the existing commenting styles in the code base, so we can be consistent?